newton/examples/example_mujoco.py

```diff
@@ -407,6 +407,41 @@ def create_model_builder(robot, num_envs, randomize=False, seed=123) -> newton.M
             nconmax=args.nconmax,
         )
 
+        # Print simulation configuration summary
+        print("\n=== Simulation Configuration ===")
+        print(
+            f"Simulating {args.num_envs} {args.robot} for {args.num_frames * example.sim_substeps} steps (dt={example.sim_dt:.6f}s)"
+        )
+        print(
+            f"CUDA graph: {example.use_cuda_graph!s}, MuJoCo CPU: {example.use_mujoco!s}, random init: {args.random_init!s}"
+        )
+        # Map MuJoCo solver enum back to string
+        solver_value = example.solver.mj_model.opt.solver
+        solver_map = {0: "PGS", 1: "CG", 2: "Newton"}  # mjSOL_PGS = 0, mjSOL_CG = 1, mjSOL_NEWTON = 2
+        actual_solver = solver_map.get(solver_value, f"unknown({solver_value})")
+        # Map MuJoCo integrator enum back to string
+        integrator_map = {
+            0: "Euler",
+            1: "RK4",
+            2: "Implicit",
+            3: "Implicitfast",
+        }  # mjINT_EULER = 0, mjINT_RK4 = 1, mjINT_IMPLICIT = 2, mjINT_IMPLICITFAST = 3
+        actual_integrator = integrator_map.get(example.solver.mj_model.opt.integrator, "unknown")
+        print(f"Solver: {actual_solver}, integrator: {actual_integrator}, actuation: {args.actuation}")
+        print(
+            f"Solver iter: {example.solver.mj_model.opt.iterations}, LS iter: {example.solver.mj_model.opt.ls_iterations}"
+        )
+        # Get actual max constraints and contacts from MuJoCo Warp data
+        actual_njmax = example.solver.mjw_data.njmax
+        actual_nconmax = (
+            example.solver.mjw_data.nconmax // args.num_envs if args.num_envs > 0 else example.solver.mjw_data.nconmax
+        )
+        print(f"Max constraints: {actual_njmax} per env, max contacts: {actual_nconmax} per env")
+        print(
+            f"Joint DOFs: {example.model.joint_dof_count}, bodies: {example.model.body_count}, device: {wp.get_device()}"
+        )
+        print("================================\n")
```

Comment:
Can we make these print statements conditional on some flag, like `verbose`? We can keep it True by default in the MuJoCoSolver constructor, but I think there should be a way to turn them off.

Reply:
NVM I didn't see this was the example code.